src/db/clients/mysql.js

```diff
@@ -127,6 +128,22 @@ export async function listTableTriggers(conn, table) {
   return data.map((row) => row.trigger_name);
 }
 
+export async function listTableIndexes(conn, table) {
+  const schema = await getSchema(conn);
+
+  const sql = `
+  SHOW INDEX FROM ${table} FROM ${schema};
```

Comment:
Could you please replace the args `${...}` with `?`. And pass all them was params in the below argument? At least is good do it for the `table` arg to avoid any sql injection.
Once you change it, there is no need for the template string and could keep it in a single line.